README.md

```diff
@@ -331,7 +295,7 @@ The secrets are available as environment variables.
 
 Please file requests via [**BugCrowd**](https://bugcrowd.com/agilebits).
 
-For information about security practices, please visit our [Security homepage](https://bugcrowd.com/agilebits).
+For information about security practices, please visit our [Security homepage]([https://bugcrowd.com/agilebits](https://1password.com/security)).
```

Comment:
I'm not sure what's happening here. 🤔 
The previous link was ok. Did you want to replace it with another one? 
```suggestion
For information about security practices, please visit our [Security homepage](https://1password.com/security).
```

Reply:
Sorry, the markdown got mangled. My bad! Wanted to update to https://1password.com/security.